awx/ui_next/src/components/Lookup/ExecutionEnvironmentLookup.jsx

```diff
@@ -26,15 +26,38 @@ function ExecutionEnvironmentLookup({
   i18n,
   isDefaultEnvironment,
   isDisabled,
+  onBlur,
   onChange,
   organizationId,
   popoverContent,
+  projectId,
   tooltip,
   value,
-  onBlur,
 }) {
   const location = useLocation();
 
+  const {
```

Comment:
I guess we needed to move it into the lookup component - that makes sense.

Reply:
I originally had this change like that, but I tried to reduce the number of props, since we want to filter using the organization id. Attempting to fetch the project before, and just pass the related org, but that generated extra requests/re-renders. Not terrible, but now the component received an extra props related to the project.